src/ray/raylet/scheduling_resources.cc

```diff
@@ -62,7 +62,7 @@ bool ResourceSet::IsSuperset(const ResourceSet &other) const {
 }
 /// Test whether this ResourceSet is precisely equal to the other ResourceSet.
 bool ResourceSet::IsEqual(const ResourceSet &rhs) const {
-  return (this->IsSubset(rhs) && rhs.IsSubset(*this));
+  return (IsSubset(rhs) && rhs.IsSubset(*this));
```

Comment:
same, I would prefer to keep `this->` for readability when overloading binary operators of the form A.BinaryOperator(B).